hermes/src/ontology/injection.rs

```diff
@@ -115,6 +126,17 @@ pub struct InjectionResetCompleteMessage {
 
 impl<'de> HermesMessage<'de> for InjectionResetCompleteMessage {}
 
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Example)]
+#[serde(rename_all = "camelCase")]
+pub struct InjectionResetFailedMessage {
+    /// The id of the `InjectionResetFailedMessage`
+    pub request_id: Option<String>,
+    /// The context of the failure
+    pub context: Option<String>,
```

Comment:
why the option there ?

Reply:
I wasn't sure we could provide a context every time. In practice we can, I'll make it non optional.